Wikipedia.xcodeproj/xcshareddata/xcschemes/Wikipedia.xcscheme

```diff
@@ -1,7 +1,7 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <Scheme
-   LastUpgradeVersion = "1430"
-   version = "1.7">
+   LastUpgradeVersion = "1500"
```

Comment:
Curious: any reason for minimum targeting 15.0 instead of a later iOS 15 version? I suppose a 15.0 minimum provides a wider install base, though maybe only marginally as anyone on iOS 15.0 should be on devices that support updating to later versions of 15. I think a lower iOS 15 minimum vs. a later one increases our support burden (though also maybe marginally), as we allow the potential for unaddressed API fixes/changes (https://developer.apple.com/documentation/ios-ipados-release-notes/ios-ipados-15_1-release-notes etc).

Reply:
These `LastUpgradeVersion` changes are the Xcode version checks - this basically prevents that "update with project recommended settings" suggestion on Xcode 15. But I did set the deployment target to [15.0](https://github.com/wikimedia/wikipedia-ios/pull/4602/files#diff-ef4712bf7dff5e1a57a3fc2a39c0397c9de3c33796d6fbfa20b9ef9d7de302ca), so your comment still stands :)

I've generally always thought that if it isn't (greatly) limiting us with what APIs we can use, we should go with the even number to increase our user base for free. But I'm open to us revisiting this! I'll follow up.

Reply:
@staykids Updated to 15.4.